src/XMakeBuildEngine/Resources/Strings.resx

```diff
@@ -615,8 +615,8 @@
     e.g. &lt;UsingTask AssemblyName="foo"&gt; -- this is missing the "TaskName" attribute.</comment>
   </data>
   <data name="IncludeRemoveOrUpdate" UESanitized="false" Visibility="Public">
-    <value>MSB4232: Static items (items not under targets) must have one of the following operations: Include, Update, or Remove.</value>
-    <comment>{StrBegin="MSB4232: "} Include, Update, and Remove should not be localized and their casing should not be changed</comment>
+    <value>MSB4232: Items that are outside Target elements must have one of the following operations: Include, Update, or Remove.</value>
+    <comment>{StrBegin="MSB4232: "} Target, Include, Update, and Remove should not be localized and their casing should not be changed</comment>
```

Comment:
If it's not an actual xml element / keyword then I wouldn't. I've checked with the other engine resources and they don't mark the term `item` as should-not-translate.